README.md

```diff
@@ -211,6 +211,8 @@ To add resources or to contribute read the [Contribution-guide](https://github.c
   
 ## Frontend
   - :video_camera: [Full Course](https://www.youtube.com/watch?v=zJSY8tbf_ys)
+  - :video_camera: [Frontend Web Development Bootcamp Course by freeCodeCamp](https://youtu.be/zJSY8tbf_ys)
+  - :hammer_and_wrench: [Roadmap by Community Classroom](https://www.commclassroom.org/roadmaps#_3iwk93nyk)
```

Comment:
Hey, Please add a link that directly redirects to the front-end roadmap.

Reply:
Yes, I've updated the link by mistake added the wrong one.